### PR TITLE
Feat: Add UI Breakpoints for Event Card

### LINKED ISSUE #116: Resolve spacing between "filter list" and "description" on card
They are too close together.

### PR DESCRIPTION
**Pull request by catheraaine**
- Adds breakpoints for Various card states
- Breakpoint needs updated for sidebar on 50/50 view

closes #116, #121, #124

### CODE REVIEW

**Comment by catheraaine** [comment]
@BC3209 @boomingbrake @iAmNathanJ

File: app/assets/stylesheets/styles/_events.scss [hunk 1]
```diff
@@ -6,47 +6,67 @@
...
   }
 
   .right {
-    width: 60%;
-    float: right;
-    clear: right;
+
+  }
+
+  @media (min-width: 62.19em) {
```
Comment (iAmNathanJ) [inline app/assets/stylesheets/styles/_events.scss]: Just curious where these measurements are coming from. 62.19em == 995.04px here I think. Is that intentional?

File: app/assets/stylesheets/styles/_events.scss [hunk 2]
```diff
@@ -150,10 +187,17 @@
     ul {
       list-style-type: disc;
       list-style-position: inside;
-      columns: 3;
-      -webkit-columns: 3;
-      -moz-columns: 3;
-      margin-top: $margin;
+      margin-top: 2em;
+      columns: 2;
```
Comment (iAmNathanJ) [inline app/assets/stylesheets/styles/_events.scss]: This is really cool, however, there are some support issues. Might need to consider an alternate approach - http://caniuse.com/#feat=multicolumn

Also, just fyi, autoprefixer should be taking care of vendor prefixes for us, so you shouldn't have to add them yourself. (awesome sauce) We should probs check our output and make sure we're covering the prefixing we need. I'll look into that.

**Comment by iAmNathanJ** [comment]
@catheraaine Check the notes on the files. Just have a couple comments / questions.

**Comment by catheraaine** [comment]
@iAmNathanJ Breakpoints are just wherever the design falls apart.

**Comment by catheraaine** [comment]
@iAmNathanJ fixed! I think.

**Comment by iAmNathanJ** [comment]
@catheraaine MERGED. Huzzah.

**Pull request merged**